framegear/components/FrameInput/FrameInput.tsx

```diff
@@ -28,12 +31,12 @@ export function FrameInput() {
         />
       </label>
       <button
-        className="h-[40px] self-end rounded-full bg-white text-black"
+        className={`h-[40px] self-end rounded-full ${isLoading ? "bg-neutral-800 text-white" : "bg-white text-black"}`}
```

Comment:
I believe you can use `clx` on Framegear as well.

Reply:
Okay, I will implement the `clx`